packages/legend-query-builder/src/stores/QueryBuilder_LegendApplicationPlugin_Extension.ts

```diff
@@ -38,6 +38,13 @@ export type DatasetEntitlementAccessReportActionConfiguration = {
   ) => React.ReactNode;
 };
 
+export type ExportOtherOption = {
+  key: string;
+  title: string;
+  icon?: React.ReactNode | undefined;
+  content: string;
```

Comment:
I foresee we would need to have more than just the text here, this should be a renderer for the tab content, think about something similar to `DataSpace` quick start